### DISCUSSION TITLE
Extend Group feature to Mute and Pan controls

### DISCUSSION THREAD

**Discussion by DominikSchaller** [jamulussoftware/jamulus#1373] [Ideas]
It would be great if the Group feature would not only work for the volume control, but also for mute and pan.

It should be done in a similar way than the volume control. So clicking mute/unmute for one client of the group is applied to all group members. For pan, the same value could either be applied to everybody, or it would work relative to the individual settings (like volume).

The SHIFT trick should still work, meaning if you press the SHIFT key you can change it for individual members of the group.

**Comment by JohannesBrx** [comment]
In general, I think this is a consequent enhancement. However, I would like to see an inverted behavior for the shift key:￼￼ Without holding shift, individual channels should be modified. By holding shift, the group properties should be changed. It is more logical for me that the more complex behavior is activated by pressing an extra key.￼￼

**Reply by dcorson-ticino-com** [reply]
> It is more logical for me that the more complex behavior is activated by pressing an extra key

But I see the complaints already that the grouping doesn't work if we have it only with shift pressed.

**Reply by DominikSchaller** [reply]
It's a problem either way. I dismissed the Group feature two months ago as useless, because I didn't know about the shift key. And I didn't want to ungroup individuals to change just their volume.

Ideally this should be solved in a different visual way. Imagine a grouping view in which each group is visualized by a border around the members of the group, an individual name (e.g., "Altos"), and group controls. An additional volume fader, pan gear, and mute button just for the group.
